easybuild/easyconfigs/n/Nipype/Nipype-1.9.2-gfbf-2024a.eb

```diff
@@ -0,0 +1,85 @@
+easyblock = 'PythonBundle'
+
+name = 'Nipype'
+version = '1.9.2'
+
+homepage = 'https://nipype.readthedocs.io'
+description = """Nipype is a Python project that provides a uniform interface to existing neuroimaging software and
+ facilitates interaction between these packages within a single workflow."""
+
+toolchain = {'name': 'gfbf', 'version': '2024a'}
+
+builddependencies = [
+    ('hatchling', '1.24.2'),
+    ('PDM', '2.18.2'),
+]
+
+dependencies = [
+    ('Python', '3.12.3'),
+    ('lxml', '5.3.0'),
+    ('networkx', '3.4.2'),
+    ('pydot', '3.0.3'),
+    ('NiBabel', '5.2.1'),
+    ('pytest-xdist', '3.6.1'),
+    ('poetry', '1.8.3'),
```

Comment:
poetry doesn't seem needed for Nipype?

Reply:
`poetry` is there to fulfil one of the extensions needed for `Nipype`. But frankly, I do not recall anymore for which extension that was needed; maybe `six` or something else.
Is that an accepted justification? Or you prefer that `poetry` be dropped and that specific extension be added explicitly to list of extensions for `Nipype`? The drawback of the latter is having duplicate (versions) of the same package in one toolchain under two different `PythonBundle`s.
But, I am open to your suggestion.

Reply:
Also looking at this (although installing against 2023a) and tried using the RDFlib dependency but it doesn't satisfy the following `prov` constraint:
```
rdflib<7,>=4.2.1
```
(See https://github.com/trungdong/prov/commit/6f7ae21f46467a0b50ee919571e9fb9aae9f1478)

I'm therefore including `rdflib` and `isodate` in the bundle but adding `poetry` to the build deps as @smoors suggests.